### PR TITLE
Mhutti1/testimprovements

### PR DESCRIPTION
**Pull request by mhutti1**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -123,17 +123,15 @@ custom.eachFile() { file ->
...
   }
   map.put(fileName, file.getAbsolutePath())
-
-
 }
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 25
```
Comment (mgautierfr) [inline app/build.gradle]: Why switching to sdk version 25 ?

Just to get the last version or there is something the sdk 25 fixes ?
(And I would have prefer to have this change in a separate PR than hidden in the lots of unrelated commits. Especially as it impacts kiwix-build)

**Comment by mgautierfr** [comment]
Please for next PR, squash all your "More attempts …"  and "Try …" commits in others.
It's ok that you use them to test and make travis working but it should no go into master.
It's complicate a lot the history and the easiness to find a specific change.

File: app/build.gradle (hunk 1 above)
Comment (mhutti1) [inline app/build.gradle]: Its a good idea to keep your compile and target SDK up to date or a new version of Android might break something. I did discuss it with you briefly, sorry for not making it more clear where the change occured.

Comment (mgautierfr) [inline app/build.gradle]: No problem, I miss this information during our discussion.

I'm agree with you, it a good idea to keep up to date with our dependencies and SDK.
However, is there any compatibility problems that could occurs by switching to sdk 25 ?
Or it is the platform who define entirely where the application could run ?

Comment (mhutti1) [inline app/build.gradle]: No, no issues like that, you need to target a certain version of the API to be able to use newer features though, (you disable them in older versions)
